ltiauthenticator/lti11/handlers.py

```diff
@@ -48,3 +48,28 @@ def post(self):
         )
 
         self.redirect(body_argument)
+
+
+class LTI11ConfigHandler(BaseHandler):
+    """
+    Renders LTI 1.1 configuration file in XML format.
+    """
+
+    async def get(self) -> None:
+        """
+        Gets the XML config which is used by LTI consumers to install the external tool.
```

Comment:
Something like `Renders` instead of `Gets` perhaps? It writes back a response to the HTTP request of GET type.

```suggestion
        Renders the XML config which is used by LTI consumers to install the external tool.
```